src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -6094,5 +6094,31 @@ public class C
 
             Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
         }
+
+        [Fact]
+        public void TestERR_ManagedAddr()
+        {
+            var comp = CreateCompilation(@"
+public unsafe struct S1
+{
+    public S1* s;
+    public object P { get => field; }
```

Comment:
Consider adding a test where this is a pointer and the other field is object.

Reply:
And flipping the types in the other struct too.

Reply:
The new test is resulting in extra binding. I haven't yet investigated why this is the case.

Reply:
It looks like the extra binding comes from the `StartPropertyType` phase. For field symbols, the constraints check was moved to the `AfterAccessorBindingChecks` call, but in property symbols constraint checks occur as part of binding the property type (SourcePropertySymbolBase.cs:1658 in this branch).

Reply:
There is no longer extra binding here.